apps/app.go

```diff
@@ -46,7 +50,7 @@ func (id AppId) String() string {
 }
 
 func (id AppId) ConsulServiceName() string {
-	return strings.Replace(strings.Trim(strings.TrimSpace(id.String()), "/"), "/", ".", -1)
+	return strings.Replace(strings.Trim(strings.TrimSpace(id.String()), "/"), "/", CONSUL_NAME_SEPARATOR, -1)
```

Comment:
Personally, I'm not sure if it's the best implementation possible (global `var` in `apps`, quite scary), maybe we should add a `separator` argument to this method and pass the configuration variable every time? Or even better, move it to Consul class and have `Consul.ServiceName(appId)`? What do you think @janisz?